GameData/RealismOverhaul/Engine_Configs/18KS7800_Config.cfg

```diff
@@ -118,23 +118,18 @@
 				key = 0.10 0.250
 				key = 0.00 0.040
 			}
+			TESTFLIGHT:NEEDS[TestLite,TestFlight]
```

Comment:
these come straight from the originals. if it's an AND, testflight configs have been broken since testlite was added, and I don't think that's the case. (and obviously I tested this at least a little bit)

Reply:
... but I do see the mix in the originals. I'll see how quick it is to fix

Reply:
Fixed (and resolved rd-0122 conflict). Turns out it was just those 2 SRBs, the very first files I converted. must have copied the NEEDS from the post-processing file (which [correctly] uses `,`)